From fc3de05dff3803f2dc651b87d2cc4166d4439f79 Mon Sep 17 00:00:00 2001
From: Cody Xie <Cody.Xie@rock-chips.com>
Date: Tue, 6 May 2014 10:13:26 +0800
Subject: [PATCH] revert this for new pll policy

Revert "rk3288-p977 : lcd-LP097Qx1 modify to change fps to 59"

This reverts commit 2563925bd6661511ba0a32e38b7b0c37bc96b793.
---
 arch/arm/boot/dts/lcd-LP097Qx1.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/lcd-LP097Qx1.dtsi b/arch/arm/boot/dts/lcd-LP097Qx1.dtsi
index 9b3ab9f93854..33626cee3d66 100644
--- a/arch/arm/boot/dts/lcd-LP097Qx1.dtsi
+++ b/arch/arm/boot/dts/lcd-LP097Qx1.dtsi
@@ -14,7 +14,7 @@
 				hactive = <2048>;
 				vactive = <1536>;
 				hback-porch = <5>;
-				hfront-porch = <50>;
+				hfront-porch = <150>;
 				vback-porch = <9>;
 				vfront-porch = <3>;
 				hsync-len = <5>;
-- 
2.35.3

